### PR TITLE
Add human-readable-time package

### PR DESCRIPTION
**Pull request by G07cha**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -378,6 +378,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [Moment.js](http://momentjs.com) - Parse, validate, manipulate, and display dates.
 - [Moment Timezone](http://momentjs.com/timezone/) - IANA Time Zone Database + Moment.js.
 - [dateformat](https://github.com/felixge/node-dateformat) - Date formatting.
+- [human-readable-time](https://www.npmjs.com/package/human-readable-time) - Date formatting by pattern.
```
Comment (sindresorhus) [inline readme.md]: Link to the GitHub repo

**Comment by sindresorhus** [comment]
How is this any better than the existing `dateformat`?

**Comment by G07cha** [comment]
It has currying support(call as constructor) and more flexible syntax

**Comment by sindresorhus** [comment]
Thanks for the suggestion, but I'm gonna pass on this.

**Pull request closed without merging**